scss/tech.scss

```diff
@@ -0,0 +1,47 @@
+.tech-container {
+  align-items: center;
+  background-color: $green;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  height: 350px;
+  width: 260px;
+
+  &::before {
+    border: 2px solid $green;
+    border-radius: 10px;
+    content: '';
+    height: 100%;
+    left: -15px;
+    position: relative;
+    top: -20px;
+    width: 100%;
+  }
+
+  .tech-logo {
+    height: 100px;
+    left: 0;
+    position: relative;
+    top: 10px;
+    width: 100px;
+  }
+  
+  .tech-title, 
+  .tech-release,
+  .tech-description {
+    margin-top: 20px;
+    font-family: 'Roboto', sans-serif;
+    font-size: 16px;
+  }
+
+  .tech-release {
+    font-weight: bold;
+    margin-bottom: 10px; 
+  }
+
+  .tech-description {
+    margin: 30px 10px;
```

Comment:
being the last element you don't need margin at the bottom, that should be handled through padding in the container. and the top margin should be handled with bottom-margin in the element above this one.

Reply:
you don't need margins at the sides, those should be handled with padding in the parent.